lib/msal-electron-proof-of-concept/samples/msal-electron-sample-app/src/Main.ts

```diff
@@ -40,15 +44,28 @@ export default class Main {
                 nodeIntegration: true,
             },
         });
+        this.mainWindow.webContents.openDevTools();
```

Comment:
what does this do?

Reply:
It opens chromium's dev tools on the window programatically (for debugging purposes). I want to keep it on during development. I'll most likely remove this line by the time Auth Code is done. Should I remove it or comment it?